### PR TITLE
Security principle

### PR DESCRIPTION
**Pull request by taniwha3**

### CODE REVIEW

**Review by orzen — APPROVED** [review]
Looks good to me :)

File: VISION.md [hunk 1]
```diff
@@ -34,6 +34,12 @@ We prefer a single function with a powerful paradigm in favor of extension and m
 We prefer organic libraries over mechanistic structure. Expose what people need, however keep the core well structured. 
 As the needs of the project grows we must carefully consider our APIs and do our best to prevent ourselves from re-creating POSIX and confusing systemcalls.
 
+#### Dialtone Security
 
+Aurae is designed to be a Distributed Operating System and act as a responsible `PID 1/init`.
 
+Security is something we should not compromise on. If Aurae is online, it must be default secure, and easy to keep secure.
```
Comment (wayneeseguin) [inline VISION.md:41]: Perhaps something like this?

"easy to keep secure by making doing the right thing either built-in or straight forward and well documented. "

Comment (taniwha3) [reply VISION.md]: updated, please let me know if it is clear to you

Comment (wayneeseguin) [reply VISION.md]: LGTM ❤️

**Pull request merged**